tests/cephadm/test_cephadm_operations_via_admin.py

```diff
@@ -44,9 +45,12 @@ def run(ceph_cluster, **kw):
             f"Failed to add label '{label}' on node '{hostname}'"
         )
     # Verify files in new admin node
-    for file in files:
-        result = os.path.exists(file)
-        if result is False:
+    timeout, interval = 300, 6
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        for file in files:
+            if os.path.exists(file):
+                break
+        if w.expired:
```

Comment:
This check should be outside of waiter loop

Reply:
Done